### DISCUSSION TITLE
TTD Example for Modals

### DISCUSSION THREAD

**Discussion by kieferjs** [livewire/livewire#5545] [1. Help]
Does anyone in this community have some phpunit case examples for livewire and modals?

**Comment by joshhanley** [comment]
@kieferjs modals are typically done on the front end using javascript, so not exactly sure what you are after. The only thing you may use is possible a `$showModal` property on the component that toggles true/false. That should be a pretty straight forward test to work out, unless I'm missing some other context you're thinking?
